doc/classes/MergeGroup.xml

```diff
@@ -0,0 +1,74 @@
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="auto_merge" type="bool" setter="set_auto_merge" getter="get_auto_merge" default="true">
+			Activates merging automatically when the [MergeGroup] enters the scene (usually during loading).
+			Alternatively you can switch this off and use [method merge_meshes] to manually activate merging.
+		</member>
+		<member name="clean_meshes" type="bool" setter="set_clean_meshes" getter="get_clean_meshes" default="false">
+			Removes degenerate triangles from meshes, which can make them more suitable for later processing, such as generating secondary UVs for lightmapping.
+			[b]Note:[/b] This step can be slow and should typically only be used when [code]baking[/code] the [MergeGroup].
+		</member>
+		<member name="convert_csgs" type="bool" setter="set_convert_csgs" getter="get_convert_csgs" default="true">
+			If [code]true[/code], [code]CSG[/code] nodes will be converted to [MeshInstance]s. These [MeshInstance]s can then be merged if suitable matches are found.
+		</member>
+		<member name="convert_gridmaps" type="bool" setter="set_convert_gridmaps" getter="get_convert_gridmaps" default="false">
+			If [code]true[/code], [GridMap]s will be converted to [MeshInstance]s. These [MeshInstance]s can then be merged if suitable matches are found.
+			[b]Note:[/b] [GridMap]s are usually rendered as [MultiMesh]es very efficiently, so converting these will often be counterproductive. Exceptions include when using the [code]GLES2[/code] backend, which can be inefficient at rendering [MultiMesh].
+		</member>
+		<member name="join_meshes" type="bool" setter="set_join_meshes" getter="get_join_meshes" default="true">
+			If [code]true[/code], as a final step, matching [MeshInstance]s can be joined by combining their surfaces to form an [i]ubermesh[/i].
+			While this is convenient, it does have the downside that all the constituent meshes will be culled as one unit, which can make culling less efficient in some situations.
+		</member>
+		<member name="max_merges" type="int" setter="set_max_merges" getter="get_max_merges" default="0">
+			When set to [code]0[/code], all matching meshes will be merged within the [MergeGroup].
+			If set to [code]1[/code] or above, only groups of a maximum of [code]max_merges[/code] meshes will be merged together. These groups will be chosen by locality. This enables getting some of the benefits of merging, while still allowing some culling to take place.
+			[b]Tip:[/b] Use [code]baking[/code] to preview what the scene will look like after merging.
+		</member>
+		<member name="min_split_poly_count" type="int" setter="set_min_split_poly_count" getter="get_min_split_poly_count" default="1024">
+			When using [code]split by locality[/code] using [member splits_horizontal] and / or [member splits_vertical], you can specify that the split will only occur for meshes above this specified poly count.
```

Comment:
`[code]` markup is meant to be used for code. If you want to emphasis normal English sub sentences you can use `[i]` I guess.